Evaluate-ModelarDB-Compression/main.py

```diff
@@ -342,12 +342,17 @@ def send_sigint_to_process(process):
                 try:
                     test_data_field_column = test_data.column(field_column_name)
                 except:
-                    # Spaces in the name may have been replaced by underscores.
-                    field_column_name_with_space = field_column_name.replace("_", " ")
-                    test_data_field_column = test_data.column(
-                        field_column_name_with_space
-                    )
-
+                    if field_column_name.__contains__(" "):
```

Comment:
Should this be `field_column_name.__contains__("_")`?

Reply:
Fixed.